### PR TITLE
added search coin functionality and improved navbar styling

### PR DESCRIPTION
**Pull request by harpreetsingh100**

### CODE REVIEW

File: src/app/components/CoinTable/CoinTable.tsx [hunk 1]
```diff
@@ -40,20 +40,18 @@ const CoinTable = () => {
...
+              color={`${theme === "light" ? "#A9AAEC" : "#6161D6"}`}
+              radius="9"
+              ariaLabel="three-dots-loading"
+              wrapperStyle={{}}
+              wrapperClass=""
+            />
+          </p>
         }>
         {coins.map((coin, i) => (
           <CoinInfoItem key={uuidv4()} coin={coin} index={i} />
```
Comment (yoloOnTheBattlefield) [inline src/app/components/CoinTable/CoinTable.tsx]: use something from the coin as a key, not uuid

File: src/app/components/SearchBar/SearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            debouncedSearchValue == ""
+              ? "hidden"
+              : "h-[420px] w-full absolute top-9 dark:bg-[#191925] bg-[#EBEBFC] z-50 overflow-y-scroll flex flex-col gap-4 transition delay-200 duration-500 py-4 pr-4 rounded-b-lg border-[#6B7280] border-[1px]"
+          }`}>
+          {data &&
+            data?.coins.map((coin: any) => {
+              return (
+                <Link
+                  href="/portfolio"
+                  key={uuidv4()}
```
Comment (yoloOnTheBattlefield) [inline src/app/components/SearchBar/SearchBar.tsx]: same here

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/app/components/CoinTable/CoinTable.tsx (hunk 1 above)
Comment (harpreetsingh100) [reply src/app/components/CoinTable/CoinTable.tsx]: fixed

File: src/app/components/SearchBar/SearchBar.tsx (hunk 2 above)
Comment (harpreetsingh100) [reply src/app/components/SearchBar/SearchBar.tsx]: fixed

**Pull request merged**